.github/ISSUE_TEMPLATE/epic.md

```diff
@@ -0,0 +1,20 @@
+---
+name: Epic
+about: A larger task consisting of more deliverables
+labels: 'Epic, priority: undecided'
+pipelines: Epics
+---
+
+## Background
+A clear and concise intro into the situation.
+
+## Goal
+The goal that epic wants to achieve.
+
+## Expected Task List [Optional]
+A list of expected issues that will be needed to achieve this Epic
+1. 
+2. 
+3. 
+
+**NB! Don't forget to assign the Epic into a Milestone**
```

Comment:
I don't think this is necessary. We have no hard requirements and deadlines.

Reply:
I mostly agree with you. What troubles me, is that we currently use milestones as a "release planner" as well. And you can have only on milestone per Epic meaning that any Epic connected to a Milestone of a theme, cannot be added to a milestone of a release.

Reply:
Do we? 😉 
We had a milestone 3.0.0 but there are literally a handful if items assigned. Also there is Spline 0.5.x and Rest API, the later probably will not get there when we finally release it. So Release based milestones are little big and stretched.
Finally we can have still major release milestones. After all the major releas changes are based on breaking features.
So to conclude, this doesn't prevent them just makes it little bit more granular, to make it more useful IMO.

Reply:
Small one: I still don't like having lines there that are made only to be deleted.